app/views/components/fields/field_accessible_autocomplete_with_header.scala.html

```diff
@@ -66,8 +66,12 @@
 
     @bodyHtml.map(identity)
 
-    @hintParagraphs.map { hint =>
-        <p class="govuk-hint" id="@(fieldNameWithoutBrackets)-hint">@hint</p>
+    @if(hintParagraphs.nonEmpty) {
```

Comment:
Why not simply...
```
   @hintParagraphs.map { hint =>
        <div id="@(fieldNameWithoutBrackets)-hint">
            <p class="govuk-hint">@hint</p>
        </div>
    }
```

Reply:
Where there are multiple paragraphs I'd just end up with each paragraph wrapped in a div with the same ID - I'm trying to wrap all the paragraphs in one div.

Reply:
Yeah, sorry my bad. I supposed `hintParagraphs` was an `Option`, and not a `List`.